internal/pkg/plugin/jiragithub/workflow.go

```diff
@@ -0,0 +1,14 @@
+package jiragithub
+
+import (
+	github "github.com/merico-dev/stream/pkg/util/github"
+)
+
+const (
+	CommitMessage   = "jira-github-integ Actions workflow, created by DevStream"
+	BuilderFileName = "jira-github-integ.yml"
```

Comment:
The "name" of the workflow (as defined in the template) is "name: Jira Github integration Builder", but the file name is "jira-github-integ.yml". I think we need better consistency here.